build.gradle

```diff
@@ -119,6 +119,10 @@ allprojects {
       url 'https://splunk.jfrog.io/splunk/ext-releases-local'
       content { includeGroupByRegex('com\\.splunk\\..*') }
     }
+    maven {
```

Comment:
This appears to be a staging repository.  How long until the full release occurs?  Or will the transition be transparent because the non-staged versions will go to central?